### PR TITLE
py-grandalf: new recipe

### PR DESCRIPTION
**Pull request by Andrew-Dunning-NNL**
add new package py-grandalf

### CODE REVIEW

**Comment by Andrew-Dunning-NNL** [comment]
This python package recipe uses github url (instead of pypi var) because the grandalf source tar is not in pypi

**Review by tldahlgren — DISMISSED** [review]
Confirmed the versions are available at the specified URL.

**Comment by tldahlgren** [comment]
@adamjstewart Do you want to review this package?

File: var/spack/repos/builtin/packages/py-grandalf/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    drawing algorithms."""
+
+    homepage = "https://github.com/bdcht/grandalf"
+    url      = "https://github.com/bdcht/grandalf/archive/v0.7.tar.gz"
+
+    version('0.7',     sha256='b3112299fe0a9123c088a16bf2f1b541d0d91199b77170a9739b569bd16a828e')
+    version('0.6',     sha256='928db4b90f7aff01e252a833951086b20d5958c00083411193c794de7bf59df2')
+
+    depends_on('python@3:', type=('build', 'run'))
+    depends_on('py-pyparsing', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-grandalf/package.py]:
```suggestion
    depends_on('py-pyparsing', type=('build', 'run'))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-grandalf/package.py (hunk 1 above)
Comment (Andrew-Dunning-NNL) [reply var/spack/repos/builtin/packages/py-grandalf/package.py]: added runtime dependency

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**